setup.py

```diff
@@ -1,18 +1,43 @@
-try:
-    from setuptools import setup
-except:
-    from distutils.core import setup
+import setuptools
 
-readme = open("README.md")
-requirements = open("requirements.txt").read().splitlines()
+def open_requirements(path):
+    with open(path) as f:
+        requires = [
+            r.split('/')[-1] if r.startswith('git+') else r
+            for r in f.read().splitlines()]
+    return requires
 
+requires = open_requirements('requirements.txt')
 
-setup(name="inference_interface",
-      version="0.1",
-      description="functions to read, convert and handle toyMC simulations, fit results and templates used by the XENONnT inference codes",
-      author="XENONnT collaboration",
-      author_email="knut.dundas.moraa@columbia.edu",
-      url="https://github.com/XENONnT/inference_interface",
-      install_requires=requirements,
-      py_modules=["inference_interface"]
-      )
+with open('README.md') as file:
+    readme = file.read()
+
+with open('HISTORY.md') as file:
+    history = file.read()
+
+setuptools.setup(
+    name='inference_interface',
+    version='0.0.0',
```

Comment:
Keep version number to 0.1.0 or above :)?

Reply:
Yes. I bumped it to `0.0.1`.